src/OpenTelemetry.ResourceDetectors.Container/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Changed .NET Targeted Frameworks to .Net's Minimum Supported Version.
```

Comment:
I'd add here. Instead of .Net Framework 4.6.2 and .Net Standard 2.0 targeted assemblies, only .Net 6.0 targeted assembly will be provided.